dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -146,10 +147,11 @@ def padField(field,value,fileType,validationInterface):
         if value is not None and validationInterface.isPadded(field,fileType):
             # If padded flag is true, get column length
             padLength = validationInterface.getColumnLength(field, fileType)
-            # Pad to specified length with leading zeros
-            return value.zfill(padLength)
-        else:
-            return value
+            if len(value) < padLength:
```

Comment:
All of the above, both user submissions and domain value loading gets run through this function.  For the above, you're right that was part of the changes for populating the padding flag, I'll simplify this function back to the earlier form.